src/Servers/Kestrel/test/InMemory.FunctionalTests/Http3/TestStreamContext.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.IO.Pipelines;
+using Microsoft.AspNetCore.Connections;
+using Microsoft.AspNetCore.Connections.Features;
+using Microsoft.AspNetCore.Http.Features;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Core.Tests
+{
+    public partial class Http3TestBase
+    {
+        internal class TestStreamContext : ConnectionContext, IStreamDirectionFeature, IStreamIdFeature, IProtocolErrorCodeFeature
```

Comment:
I'm not in love with splitting Http3TestBase into a bunch of files. This could just not be nested inside Http3TestBase since it's internal not private.

Reply:
Ah, oops, didn't realize VS refactoring left the partial classes everywhere.